### PR TITLE
chore: Added unit tests for FileSystemUndeleteHandler

### PR DESCRIPTION
**Pull request by derektriley**
**Description**:
Adds unit tests for FIleSystemUndeleteHandler. Brings coverage to 92%.

**Related issue(s)**:
#13193 

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileSystemUndeleteTest.java [hunk 1]
```diff
@@ -21,19 +21,23 @@
 import static com.hedera.hapi.node.base.ResponseCodeEnum.UNAUTHORIZED;
 import static com.hedera.node.app.spi.fixtures.Assertions.assertThrowsPreCheck;
 import static com.hedera.test.utils.KeyUtils.A_COMPLEX_KEY;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
```
Comment (thomas-swirlds-labs) [inline hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileSystemUndeleteTest.java:24]: Favour AssertJ over Junit Jupiter Assertions

Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileSystemUndeleteTest.java:24]: I agree with Thomas better to use AssertJ

**Review by povolev15 — APPROVED** [review]
Other than @thomas-swirlds-labs comment looks good @derektriley

**Review by thomas-swirlds-labs — APPROVED** [review]
LGTM. Thanks! @derektriley 🎉

**Review by povolev15 — APPROVED** [review]
LGTM TY @derektriley

**Pull request merged**